src/vos/vos_iterator.c

```diff
@@ -609,6 +665,39 @@ vos_iter_cb(vos_iter_cb_t iter_cb, daos_handle_t ih, vos_iter_entry_t *iter_ent,
 	return rc;
 }
 
+static inline bool
+is_skip(unsigned int acts)
+{
+	return (acts & VOS_ITER_CB_SKIP);
+}
+
+static inline bool
+is_delete(unsigned int acts)
+{
+	return (acts & VOS_ITER_CB_DELETE);
+}
+
+#define JUMP_TO_STAGE(rc, next_label, probe_label, abort_label)				\
+	do {										\
+		switch (rc) {								\
+		case ITER_ABORT:							\
+			(rc) = 0;							\
+			/* fallthrough */						\
+		case ITER_EXIT:								\
+			goto abort_label;						\
+		case ITER_NEXT:								\
+			goto next_label;						\
+		case ITER_PROBE:							\
+			goto probe_label;						\
+		case ITER_CONTINUE:							\
+			(rc) = 0;							\
+			break;								\
+		default:								\
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment